### DISCUSSION TITLE
Welcome to Praxel.ai Community Discussions!

### DISCUSSION THREAD

**Discussion by praxelai** [praxelai/community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. Please see our [FAQ](https://praxel.ai/about#faq) first as we may have already answered your question there.
  * Share ideas and features you would like to see on our product.
  * Engage with other community members.
  * Report bugs you find on the platform.
  * **IMPORTANT**, for any bug you consider to be a security bug, please do not post on this forum. Kindly send an email to **support@praxel.ai** and we will follow up with you as required. Kindly note, we do not provide bounties for bug reports.

  To get started, comment below with an introduction of yourself (e.g. your technology interests) and tell us about what you plan to **Practice** and **Excel** at on [Praxel.ai](https://praxel.ai)

**Comment by Wunmi93** [comment]
I have knowledge of programming in my journey from being a fresher. I'm interested in a career in technology because I want to use my skills and knowledge to make other people's jobs easier and more productive. I believe IT support technicians/programmers are important team members for any organization because they combine technical knowledge with interpersonal skills.

**Comment by AbdullaahAlqaasim** [comment]
Hello, I am @AbdullaahAlqaasim ,
 I am a fullstack developer with experience in Data Analysis and IT automation. I’d be happy to help you navigate the[ Praxel.ai](http://praxel.ai/) platform and escalate any issues you may encounter to the appropriate internal teams. Please feel free to @mention me in your discussions. Practice and Excel!
